methods/userEvents.coffee

```diff
@@ -61,3 +61,8 @@ Meteor.methods
         $unset:
           lastIncidentDate: ""
       })
+
+if Meteor.isServer
```

Comment:
It seemed link an intuitive place to put all the meteor methods.  So this folder is only for `dual-purpose` code, e.g both client and server?